examples/common_vmseries/Makefile

```diff
@@ -1,32 +1,15 @@
-init:
-	@../../makefile.sh init
-
-prep_vars: check_uuid
-	@if [ ! -f ghci.tfvars ]; then sed -E "s/example-/${PREFIX}/g;s/^resource_group_name.*/resource_group_name = \"${RG}\"/g" example.tfvars > ghci.tfvars; fi
-
-validate:
-	@../../makefile.sh validate
-
-plan: init prep_vars
-	@../../makefile.sh plan
-
-apply: init prep_vars
-	@../../makefile.sh apply
-
-idempotence:
-	@../../makefile.sh idempotence
-
-destroy:
-	@../../makefile.sh destroy
-	@rm ghci.tfvars
-
-check_uuid:
-ifndef UUID
-	$(info Missing UUID, generate one for yourself using command:)
-	$(info export UUID=$$(uuidgen | tr '[:upper:]' '[:lower:]'))
-	$(error )
-else
-RG := $(shell echo ${UUID} | cut -d '-' -f 1,5)
-PREFIX := ghci$(shell echo ${UUID} | cut -d '-' -f 2)-
-STORAGE := $(shell echo ${UUID} | cut -d '-' -f 2,3,4 | tr -d '-')
-endif
\ No newline at end of file
+init_go:
+	@echo "::group::DOWNLOADING GO DEPENDENCIES" && \
```

Comment:
Previously we introduced `makefile.sh` in order to in have in every `Makefile` only 1 command for 1 action e.g. for AWS VM-Series modules we have in `Makefile`:

```
test:
	@../../makefile.sh test
```

and in `makefile.sh`:

```
  test)  
    echo "::  DOWNLOADING GO DEPENDENCIES  ::"
    go get -v -t -d && go mod tidy
    echo
    echo "::  EXECUTING TERRATEST  ::"
    go test -v -timeout 120m -count=1
    echo
    ;;
```

Now after removing `makefile.sh`, we need to repeat 3 lines for every example e.g.:

```
	@echo "::group::TESTING INFRASTRUCTURE" && \
	go test -run Deploy -timeout 60m -count=1 && \
	echo "::endgroup::"
```

Why we are removing `makefile.sh` ?

Reply:
I think we can, I'm trying to figure out if we can have perhaps a single makefile?

Reply:
and previously makefiles where made of two parts, one that was common and one that was specific to a module (namely setting up unique variables).
with Terratest the latter one was moved to `main_test.go`, leaving the makefile quite generic. It's easier just to copy it between modules w/o any changes to itself.

Reply:
If there is a chance to not repeat the same code by copying files, I prefer to do it. Nevertheless it's better to have consistent approach through all clouds for VM-Series modules. Are we then removing `makefile.sh` from all repositories ?

Reply:
> If there is a chance to not repeat the same code by copying files...
you will still copy them, they are still the same, the content is different though, meaning they originally contained a reference to `makefile.sh`, now they just have 3~4 lines per target.

My point is that when we had complex logic in `Makefile`'s, a lot of variables, etc it was easier to move the code to a separate file (especially that all code in a `Makefile` is a oneliner, you can't control `set -e` and `set -o pipefail` that easily, it's harder to operate on env variables, etc).

But now the logic is moved to `main_test.go` and I do not see a point in keeping a `makefile.sh` in this situation as the `Makefile`'s code is way more simpler, not parametrized and we no longer need to control execution env to use for example `grep` w/o failing the whole pipeline.

Reply:
should we use the same approach in all repos? I do not know TBH, the `Makefile` is a common abstraction layer, but what it contains does not have to be IMO.